test.sh

```diff
@@ -1,19 +1,9 @@
 #!/usr/bin/env bash
 
-ORIGIN=https://hubbed:$GITHUB_TOKEN@github.com/williamkapke/node-compat-table.git
+curl -fsSL https://fnm.vercel.app/install | bash -s -- --install-dir "./.fnm" --skip-shell
```

Comment:
why are we using fnm here? if we need multiple node versions, let's just use my shared nvm action and use nvm.

Reply:
Do you have a link?

The problem is that we need to dynamically iterate over a big list and grab all of them, but remove then as to avoid filling up the disk. That doesn't sound like something a GitHub Action can do.

Reply:
It's something nvm can do, which is a far more widely used tool than fnm.

Reply:
I am not strongly attached to `fnm` besides being familiar, but I'm still looking for your `nvm` setup action.

Reply:
I've posted review suggestions that should get it mostly working.

Reply:
I'm not sure why that's valuable, but certainly would like to hear @williamkapke's thoughts.